Makefile

```diff
@@ -1,6 +1,7 @@
 .DEFAULT_GOAL:=install
 
 SOURCE_FILES=spectree tests examples
+MYPY_SOURCE_FILES=spectree tests # temporary
```

Comment:
This is here since examples are failing mypy check with pydantic v2 installed. Should they be updated to use v2?

Reply:
LGTM. We can do it later.